src/AWS.Deploy.Common/Recipes/RecipeDefinition.cs

```diff
@@ -108,6 +108,11 @@ public class RecipeDefinition
         /// </summary>
         public bool PersistedDeploymentProject { get; set; }
 
+        /// <summary>
+        /// The recipe ID of the base recommendation using which the persisted deployment project is created.
```

Comment:
Some grammar issues in this doc makes it hard to follow what this is. I assume it should be something like "The recipe ID of the recipe a saved deployment project was created from."

Reply:
Fixed